lib/src/soloud.dart

```diff
@@ -1333,6 +1350,57 @@ interface class SoLoud {
     _controller.soLoudFFI.setFftSmoothing(smooth);
   }
 
+  /////////////////////////////////////////
+  /// voice groups
+  /////////////////////////////////////////
+
+  /// Used to create a new voice group. Returns 0 if not successful.
```

Comment:
In Dart, failure that likely has effect on the following code (like here) should generally throw. Alternatively, return `null`. In both cases, the developer is forced to deal with the possibility of failure (fail fast) instead of mistakenly trudging on.

Reply:
Right! I'll throw a new exception.